odoo/addons/base/models/ir_fields.py

```diff
@@ -200,6 +202,15 @@ def _str_to_json(self, model, field, value):
             msg = _("'%s' does not seem to be a valid JSON for field '%%(field)s'")
             raise self._format_import_error(ValueError, msg, value)
 
+    def _str_to_properties(self, model, field, value):
+        try:
+            properties = ast.literal_eval(re.sub(r"'(\b(?:True|False)\b)'", lambda m: m.group(1).lower(), value))
```

Comment:
Why you are not using json.loads (it avoids to do the regex part) ?

Reply:
json.loads does not work because the values are in single quotes and also the values of boolean are: True/False which is not supported for json.loads. See: [{'name': 'ebc5ae6962a80a96', 'tags': [['aaa', 'aaa', 8], ['bbb', 'bbb', 9], ['ccc', 'ccc', 10]], 'type': 'tags', 'string': 'demo', 'default': False, 'value': ['aaa', 'bbb']}]

Reply:
> Why you are not using json.loads (it avoids to do the regex part) ?

json.loads does not work because the values are in single quotes and also the values of boolean are: True/False which is not supported for json.loads. See: [{'name': 'ebc5ae6962a80a96', 'tags': [['aaa', 'aaa', 8], ['bbb', 'bbb', 9], ['ccc', 'ccc', 10]], 'type': 'tags', 'string': 'demo', 'default': False, 'value': ['aaa', 'bbb']}]

Reply:
IMHO, it is not correct:
- The regex can match inside the data itself, which isn't correct: `re.sub(r"'(\b(?:True|False)\b)'", lambda m: m.group(1).lower(), str({"value": "bla'True'blu"}))` -> `blatrueblu`
- What about None?

Properties is not a Python `dict`, it should be a JSON. The exportation should return a JSON (as string, with `json.dumps`) and the import should reverse this string with a `json.loads`. What do you think about that ?

Also, can you add a test ?